src/acl.c

```diff
@@ -3042,7 +3047,7 @@ void aclCommand(client *c) {
             switch (le->context) {
             case ACL_LOG_CTX_TOPLEVEL: ctxstr = "toplevel"; break;
             case ACL_LOG_CTX_MULTI: ctxstr = "multi"; break;
-            case ACL_LOG_CTX_LUA: ctxstr = "lua"; break;
+            case ACL_LOG_CTX_SCRIPT: ctxstr = le->scripting_engine_name; break;
```

Comment:
Do we really need to show the scripting engine name here? For modules, we don't show the module name. We just show "module".

We could show just "script", but for backward compat reasons we should show "lua" for Lua scripts.

How about using "lua" for Lua scripts and "script" for other engines? If we do this, then we don't need to store the engine name in the ACL log. We can just use two different contexts:

```suggestion
            case ACL_LOG_CTX_LUA: ctxstr = "lua"; break;
            case ACL_LOG_CTX_SCRIPT: ctxstr = "script"; break; /* non-Lua script */
```